uri-schemes.md

```diff
@@ -124,6 +126,34 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+DCLOGIN:minVersion/CredentialsJSON
+# example:
+DCLOGIN:1/{"email":"me@example.com","password":""}
```

Comment:
entirely nitpicking and i have no idea what the established conventions are for QR codes, but I'd keep the `:` separator instead of having two separators: `DCLOGIN:1:{"hello": "json"}`

It means splitting the part is getting 3 chunks of the same separator starting from the front.

Reply:
for our qr-code it is that they should be URI-like.

I just moved the version to the back behind a hashtag, to make a possible future uri based format possible:
```
dclogin://password:name@example.com?smtpserver=loginname:password@smtp.example.com:1234&imapserver=loginname:password@imap.example.com:5678&smtpsecurity=ssltls&imapsecurity=ssltls&authmethod=auto&certchecks=auto
```

as suggested by [974F7BB1D254](https://support.delta.chat/u/974F7BB1D254) in the forum: https://support.delta.chat/t/allow-users-to-generate-qr-codes-to-simplify-the-login-process/1406

but I don't want to deal with uri escaping stuff for the first version I decided to use json because support for it is already in the core and it is simple.

Reply:
i see, i'm not sure if aiming for a URI format is that interesting.  what's the maximum size to use in a qrcode?

Reply:
depending on the error correction level the max size ranges from 1.273kB to 2.953kB

Reply:
thinking about it maybe it makes sense to go straight to sth like base64 encoded CBOR, because JSON might need uri escaping encoding and decoding anyway?